docs/content/envoy-authorization.md

```diff
@@ -0,0 +1,334 @@
+---
+title: Envoy Authorization
```

Comment:
With #1687 we no longer have words like "authorization" or "admission control" in the title. Let's update this to be just "Envoy".

Reply:
Done.